job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/ExtendedEmailTriggerContext.groovy

```diff
@@ -88,4 +89,11 @@ class ExtendedEmailTriggerContext implements Context {
     void replyToList(String... replyToList) {
         this.replyToList.addAll(replyToList)
     }
+
+    /**
+     * Specifies the failure count to be applied to the xNthFailure use case.
+     */
+    void failureCount(int failureCount) {
+        this.failureCount = failureCount
+    }
```

Comment:
The failure count for `XNthFailureTrigger` applies only to `XNthFailureTrigger` and not to other triggers. Therefore, it should be in its own context that extends `ExtendedEmailTriggerContext` so that it does not apply to other triggers.

Reply:
Ah, good call, that's a much better idea!  I'll adjust